scripts/schema/config/boundaries.json

```diff
@@ -383,5 +383,25 @@
         }
       }
     }
+  },
+  "$defs":
+  {
+    "Direction":
+    {
+      "anyOf":
+      [
+        {
+          "type": "string",
+          "enum": ["R", "X", "Y", "Z", "+R", "+X", "+Y", "+Z", "-R", "-X", "-Y", "-Z",
+                   "r", "x", "y", "z", "+r", "+x", "+y", "+z", "-r", "-x", "-y", "-z"]
```

Comment:
After discovering this trick, it's probably worth applying to all the other enum inputs for the config files.